doc/examples/hybrid_planning/hybrid_planning_tutorial.rst

```diff
@@ -36,7 +36,7 @@ To start the hybrid planning demo simply run: ::
 
 You should see a similar behavior as in the example GIF above without the replanning.
 
-To interact with the architecture you simply need to send a *Hybrid Planning Request* to an action server offered by the *Hybrid Planning Manager* as seen in the :moveit_codedir:`hybrid_planning_test_node <moveit_ros/hybrid_planning/test/hybrid_planning_demo_node.cpp#L245>`.
+To interact with the architecture you simply need to send a *Hybrid Planning Request* to an action server offered by the *Hybrid Planning Manager*.
```

Comment:
@sjahr This file was removed from the `moveit2` repo last year with the promise that it would eventually be moved to this repo with a refresh of the Hybrid Planning tutorials. I assume this whole file would then get revisited.